kogito-infinispan-persistence-quarkus/README.md

```diff
@@ -129,3 +170,6 @@ Last but not least you can complete review user task by
 ```
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"review" : "very good work"}' http://localhost:8080/dealreviews/xxx-aaa-cccc/review/xxx-yyy-zzz
 ```
+
```

Comment:
line 161
- we should use `{uuid}` and `{tuuid}` placeholders instead of `xxx-aaa-cccc` and `xxx-yyy-zzz`
- `curl -H 'Content-Type:application/json' -H 'Accept:application/json' http://localhost:8080/dealreviews/uuid/review/{tuuid}?user=john`

Reply:
Hi @radtriste Done